exercises/connect/src/test/java/ConnectTest.java

```diff
@@ -0,0 +1,133 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class ConnectTest {
+
+    @Test
+    public void testEmptyBoard() {
+        String[] board = new String[]{
+                ". . . . .",
+                " . . . . .",
+                "  . . . . .",
+                "   . . . . .",
+                "    . . . . ."
+        };
+        assertThat(new Connect().findWinner(board), is(""));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testOnlySmallX() {
```

Comment:
Please check that the test method names align with the descriptions provided in the canonical data.